### DISCUSSION TITLE
Installation instructions

### DISCUSSION THREAD

**Discussion by alanjonesit** [FelixKratz/SketchyBar#188] [Q&A] [answered]
Hi folks. Don't know if it's just me, but I'm having trouble installing SketchyBar. Followed the instructions at https://felixkratz.github.io/SketchyBar/setup but I'm having no luck. I feel like the instructions to install could be improved? I did the brew install instructions, now what?

**Comment by FelixKratz** [comment, marked as answer]
If you have followed the instructions closely and have disabled the default macOS bar (settings -> Dock & Menu Bar -> Automatically hide and show the menu bar) you should see the default config of sketchybar at the top of the screen once you run the brew service. I am not sure at which step in the instructions you had problems? What do you mean with "no luck"?

**Reply by alanjonesit** [reply]
I somehow missed the hide and show the menu bar instructions in your step, that's my bad, apologies.

This looks like an interesting tool, looking forward to diving into it more. I do think the setup process could be improved with some numbered steps, screenshots and more detail, but that's my two cents. Appreciate your work and reply, thank you.

**Reply by FelixKratz** [reply]
I agree with you that the setup should be as clear and precise as possible. Since I have created the instructions myself, I find them easy enough to follow, but that is clearly a biased opinion since I have spent so much time with this already. It might be very helpful to get some more precise feedback so I can improve the instructions, possibly even a layout of what you think would be easier to follow.

I could create a "one line installer", but since this is a tool targeted towards advanced users, I feel that might be inappropriate.
